scout/server/blueprints/variants/views.py

```diff
@@ -126,6 +125,8 @@ def variants(institute_id, case_name):
     if request.form.get("export"):
         return controllers.download_variants(store, case_obj, variants_query)
 
+    show_dismiss_block = controllers.get_show_dismiss_block()
```

Comment:
These are the new ones..

Reply:
These files are heavily duplicated, but perhaps avoid these lines by calling directly controllers.get_show_dismiss_block() later, when you return the data?

Reply:
Sure, it's not liket they get modified more!